docs/deploying-airbyte/on-gcp-compute-engine.md

```diff
@@ -45,7 +45,10 @@ gcloud init # Follow instructions
 
 {% tab title="Ubuntu" %}
 ```bash
-# FIXME
+echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
```

Comment:
i'm assuming you've tested these

Reply:
Yep I have a machine with Ubuntu on it, I might use it to update all of our docs with instructions for Ubuntu.